src/test/scala/wdl4s/SyntaxErrorSpec.scala

```diff
@@ -256,5 +256,17 @@ class SyntaxErrorSpec extends FlatSpec with Matchers {
         |}
       """.stripMargin)
   }
+  it should "detect when expressions within a command block don't reference actual variables" in {
+    expectError(
+      """task a {
+        |  Int x
+        |  command { ./script ${x+y} }
+        |}
+        |
+        |workflow w {
+        |  call a
```

Comment:
Actually, does assigning to a variable that's never declared also constitute a syntax error?